### PR TITLE
RAD-272 Refactor mailer tests and add bootstrap file

### PR DESCRIPTION
**Pull request by b-dabrowski**
Clear emails queue before each test

### CODE REVIEW

File: services/mailer/src/integration-tests/send-grid-transport.spec.ts [hunk 1]
```diff
@@ -43,4 +46,9 @@ describe("SendGrid transport tests", () => {
 
     return request(app).post("/api/mailer/send").send(body).expect(CREATED);
   }).timeout(15000);
+
+  after(async () => {
+    const { container } = GLOBAL.bootstrap;
+    container.register("mailerConfig", asValue({ ...originalMailConfig }));
+  });
```
Comment (eMKoded) [inline services/mailer/src/integration-tests/send-grid-transport.spec.ts:53]: Do we really need this?

File: services/mailer/src/integration-tests/smtp-transport.spec.ts [hunk 2]
```diff
@@ -45,8 +48,16 @@ describe("SMTP transport tests", () => {
...
+
     const body = {};
 
     return request(app).post("/api/mailer/send").send(body).expect(BAD_REQUEST);
   });
+
+  after(async () => {
+    const { container } = GLOBAL.bootstrap;
+    container.register("mailerConfig", asValue({ ...originalMailConfig }));
+  });
```
Comment (eMKoded) [inline services/mailer/src/integration-tests/smtp-transport.spec.ts:62]: Do we really need this?

File: services/mailer/src/integration-tests/send-grid-transport.spec.ts (hunk 1 above)
Comment (b-dabrowski) [reply services/mailer/src/integration-tests/send-grid-transport.spec.ts]: @eMKoded you have right we don't need it in this case. I have changed that.

File: services/mailer/src/integration-tests/smtp-transport.spec.ts (hunk 2 above)
Comment (b-dabrowski) [reply services/mailer/src/integration-tests/smtp-transport.spec.ts]: @eMKoded you have right we don't need it in this case. I have changed that.

File: services/mailer/src/integration-tests/bootstrap.ts [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+beforeEach("Clean email queue", async function () {
+  const { redisPrefix, redisUrl } = global.getBootstrap();
+
+  const redisClient = createClient({
+    url: redisUrl,
+    prefix: redisPrefix,
+  });
+
+  redisClient.flushall();
```
Comment (mkopa) [inline services/mailer/src/integration-tests/bootstrap.ts:47]: FLUSHALL method deleting all keys from every databases. You can use the FLUSHDB method instead.

Comment (b-dabrowski) [reply services/mailer/src/integration-tests/bootstrap.ts]: @mkopa thanks for pointing that. I have changed it to `del()`

**Pull request merged**